addons/account/models/chart_template.py

```diff
@@ -301,12 +301,20 @@ def _pre_reload_data(self, company, template_data, data):
         current_taxes = self.env['account.tax'].with_context(active_test=False).search([
             *self.env['account.tax']._check_company_domain(company),
         ])
+
+        current_fiscal_positions =  self.env['account.fiscal.position'].with_context(active_test=False).search([
+            *self.env['account.fiscal.position']._check_company_domain(company),
+        ])
         unique_tax_name_key = lambda t: (t.name, t.type_tax_use, t.tax_scope, t.company_id)
         unique_tax_name_keys = set(current_taxes.mapped(unique_tax_name_key))
         xmlid2tax = {
             xml_id.split('.')[1].split('_', maxsplit=1)[1]: self.env['account.tax'].browse(record)
             for record, xml_id in current_taxes.get_external_id().items() if xml_id.startswith('account.')
         }
+        account_fiscal_positions= {
```

Comment:
`account_fiscal_positions` doesn't say what this is. I would guess that it is just a recordset.
```suggestion
        xmlid2fiscal_position = {
```